### PR TITLE
fix for a memory leak

### PR DESCRIPTION
**Pull request by AliKarpuzoglu**
removed the constant creation of new `<canvas> ` that would spike memory usage and also added a missing semicolon

### CODE REVIEW

**Comment by AliKarpuzoglu** [comment]
@googlebot I signed it!

File: snippets/markdown/image/tensorflowjs/p5js.md [hunk 1]
```diff
@@ -50,6 +50,8 @@ Open up the code snippet below directly in the [p5.js Web Editor](https://editor
   function classifyVideo() {
     flippedVideo = ml5.flipImage(video)
     classifier.classify(flippedVideo, gotResult);
+    flippedVideo.remove();
```
Comment (shiffman) [inline snippets/markdown/image/tensorflowjs/p5js.md]: Oh, thanks for catching this! @joeyklee I wonder if there is a more elegant way for us to handle this on the ml5 side?

Comment (joeyklee) [reply snippets/markdown/image/tensorflowjs/p5js.md]: @AliKarpuzoglu - thank you for catching this. 
@shiffman - Most definitely. I can take a look at this.

**Comment by Samlilli** [comment]
Will this be merged in? Still open but approved!

**Comment by HalfdanJ** [comment]
Thanks for reminding us! Slipped through the cracks, it's merged now.

**Comment by Samlilli** [comment]
@HalfdanJ - I noticed there's another PR that's open from @AliKarpuzoglu. Seems like a legitimate ask re image normalisation, could that be checked too?

**Pull request merged**